tests/model_serving/model_server/private_endpoint/constants.py

```diff
@@ -0,0 +1,33 @@
+from typing import Dict, List, Any
+
+AWS_REGION: str = "us-east-2"
+AWS_BUCKET: str = "ods-ci-wisdom"
+AWS_ENDPOINT: str = "https://s3.us-east-2.amazonaws.com/"
+
+SR_CONTAINERS_KSERVE_CAIKIT: List[Dict[Any, Any]] = [
+    {
+        "args": ["--model-name=/mnt/models/artifacts/"],
+        "command": ["text-generation-launcher"],
+        "env": [{"name": "HF_HOME", "value": "/tmp/hf_home"}],
+        "image": "quay.io/modh/text-generation-inference@sha256:294f07b2a94a223a18e559d497a79cac53bf7893f36cfc6c995475b6e431bcfe",
```

Comment:
the image can be taken from process SR template

Reply:
check the code for pvc test  for Serving runtime u can pull the same function here
we dont have to add the  runtime image

Reply:
If you mean https://github.com/opendatahub-io/opendatahub-tests/blob/main/tests/model_serving/model_server/storage/pvc/conftest.py#L132, I had looked at it but I'm not convinced it would work correctly. 
It is passing `kserve-ovms` as a template name (https://github.com/opendatahub-io/opendatahub-tests/blob/main/tests/model_serving/model_server/storage/constants.py#L10C23-L10C33) but I can't see where that template is actually defined, either in https://github.com/opendatahub-io/opendatahub-tests/blob/main/utilities/serving_runtime.py#L8 or in https://github.com/RedHatQE/openshift-python-wrapper/blob/main/ocp_resources/template.py#L1

Reply:
https://github.com/opendatahub-io/opendatahub-tests/blob/main/utilities/serving_runtime.py#L17

Reply:
here is the template being fetch based on name

Reply:
Then I'm not sure I understand the logic here, is it only creating a `Template` instance with name `kserve-ovms` and namespace `redhat-ods-applications`? Where does it fetch the actual ServingRuntime details needed to instance the CR?

Reply:
Oh, that's cool! I didn't know about this.